zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinCore.thrift

```diff
@@ -40,7 +40,6 @@ struct Annotation {
   1: i64 timestamp                 # microseconds from epoch
   2: string value                  # what happened at the timestamp?
   3: optional Endpoint host        # host this happened on
-  4: optional i32 duration         # how long did the operation take? microseconds
```

Comment:
can we keep it here and just document as deprecated?  I'm concerned this would make historical data in Cassandra incompatible.